docs/source/_toctree.yml

```
@@ -38,6 +38,22 @@
           title: ClickHouse
         - local: duckdb
           title: DuckDB
```

The docs preview doesn't seem to work ? this might fix it
```suggestion
        - title: DuckDB
```